force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +89,17 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+    List<String> words = sentence.split(' ');
         // Create a set to store the unique words
-
+    Set<String> uniqueWords = new Set<String>(words);
         // Add each word to the set
-
+    for (String word : words) {
+        uniqueWords.add(word);
+    }
         // Join the unique words back into a sentence
-
+        String result = String.join(new List<String>(uniqueWords), ' ');
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Good work here.  A pro tip is you can pass a List directly into a Set constructor  and there is no need to iterate over anything. 

```
  public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```